web/core/store/user/base-permissions.store.ts

```
@@ -2,69 +2,70 @@ import set from "lodash/set";
     workspaceSlug?: string,
     projectId?: string,
     onPermissionAllowed?: () => boolean
   ) => boolean;
-  // action helpers
   // actions
   fetchUserWorkspaceInfo: (workspaceSlug: string) => Promise<IWorkspaceMemberMe | undefined>;
   leaveWorkspace: (workspaceSlug: string) => Promise<void>;
-  fetchUserProjectInfo: (workspaceSlug: string, projectId: string) => Promise<IProjectMember | undefined>;
+  fetchUserProjectInfo: (workspaceSlug: string, projectId: string) => Promise<TProjectMembership | undefined>;
   fetchUserProjectPermissions: (workspaceSlug: string) => Promise<IUserProjectsRole | undefined>;
   joinProject: (workspaceSlug: string, projectId: string) => Promise<void | undefined>;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)